src/ui.dropdown-badge/UDropdownBadge.vue

```diff
@@ -54,7 +54,6 @@
       ref="dropdownListRef"
       :size="size"
       :options="options"
-      :value-key="valueKey"
```

Comment:
why `valueKey` has been removed?

Reply:
it was needed for correct model value update, we relay on click-option event now so it's not required anymore